RELEASE_NOTES.md

```diff
@@ -7,6 +7,7 @@ This file contains a high-level description of this package's evolution. Release
 
 ## 4.10.0 - TBD
 
+* Modify the way xarray attribute sets are handled. See [Github ????](https://github.com/Unidata/netcdf-c/issues/????) for more information. 
```

Comment:
@DennisHeimbigner Reviewing this, do you remember what the issue is? I can go look it up if need be.